src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -28,7 +28,9 @@
             "publishBuildPlanUrl": "Publish Build Plan",
             "allowOnlineEditor": "Allow Online Editor",
             "programmingLanguage": "Programming Language",
-            "packageName": "Package Name"
+            "packageName": "Package Name",
+            "templateResult": "Template Result",
+            "solutionResult": "Solution Result"
```

Comment:
please provide proper German translations

Reply:
@krusche I tried, but it seems that we don't have consistent translations here:
'Result' and 'Template' are currently not translated at all, and 'Solution' sometimes is not translated, sometimes it is translated as 'Lösung' and once even as 'Musterlösung'.

I guess we should select one translation for each of these words and then translate all occurrences in the same way to keep consistence.

Reply:
yes, they are unfortunately missing in a couple of places. Feel free to translate everything that is not yet translated and try to make it as consistent as possible. I would go for 'Lösung' because 'Musterlösung' sounds too much as there would only be one solution. We might go for 'Beispiel Lösung' later. 'template' should be 'Vorlage', 'Result' should be 'Ergebnis'

Reply:
I've translated it, i'm not really happy with it but at least it should be consistent now.